docs/index.bs

```diff
@@ -62,6 +64,8 @@ spec: ecma-262; urlPrefix: http://tc39.github.io/ecma262/
     type: abstract-op
         text: NormalCompletion; url: sec-normalcompletion
         text: ThrowCompletion; url: sec-throwcompletion
+        text: IsCallable; url: sec-iscallable
+        text: Get; url: sec-get-o-p
```

Comment:
These should not be necessary these days. But, maybe they are; if removing them breaks then ignore my advice :)

Reply:
They seem to be needed. 
LINE ~3011: No 'abstract-op' refs found for 'IsCallable'.
[$IsCallable$]
LINE ~3012: No 'abstract-op' refs found for 'Get'.
[$Get$]
LINE ~3015: No 'abstract-op' refs found for 'IsCallable'.
[$IsCallable$]